### PR TITLE
Requiring build_id to deploy

### PR DESCRIPTION
**Pull request by tommie24**

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
+if [[ -z ${DEPLOYBOT_SERVICE_URL} || -z ${SERVICE_NAME} || -z ${COMMIT_SHA} || -z ${BUILD_ID} ]];
 then
-  echo "Each of these variables are required: DEPLOYBOT_SERVICE_URL, SERVICE_NAME and COMMIT_SHA"
+  echo "Each of these variables are required: DEPLOYBOT_SERVICE_URL, SERVICE_NAME, COMMIT_SHA and BUILD_ID"
   exit
 fi
 
-curl "${DEPLOYBOT_SERVICE_URL}/${SERVICE_NAME}" -H "Content-Type: application/json" -d '{"hash":"'"${COMMIT_SHA}"'"}'
\ No newline at end of file
+curl "${DEPLOYBOT_SERVICE_URL}/${SERVICE_NAME}" -H "Content-Type: application/json" -d '{"hash":"'"${COMMIT_SHA}"'", "build_id":"'"${BUILD_ID}"'"}'
```
Comment (normrider2001) [inline run.sh]: Probably should add a new line at end of file.

**Review by normrider2001 — APPROVED** [review]

**Comment by normrider2001** [comment]
I'm not sure how to publish this.  Might just be a tag.

**Comment by tommie24** [comment]
> I'm not sure how to publish this. Might just be a tag.

How do I add the tag?

**Comment by normrider2001** [comment]
I think you have to draft a release.  The release should be V2.

**Pull request merged**